### DISCUSSION TITLE
Feature ideas

### DISCUSSION THREAD

**Discussion by Alexiscomete** [Sellig6792/FuckBF#3] [Ideas]
I have many ideas for "FuckBrainFuck"

Should I create one discussion per idea or a general discussion with all ideas?

The task list of all my ideas (to discuss about them, and write the result)

# Functions

- [x] Can we have a temporary function into another function?
- [ ] Set a char for a function to have multiple functions

# Memory

- [ ] insert a new memory case before the current case (for functions)
- [ ] remove the current case and set the current case to the next case / remove the last case the keep the "1 char -> 1 utility"

# Instant value

- [ ] A value is given by a char in the current case (replace)

OR if the memory is accepted : 

- [ ] A value in a new case is given by a char

## Values

- [ ] get the current case

I will update this message

**Comment by Alexiscomete** [comment]
What do you think about ?

**Reply by gilles-mnr** [reply]
Some of these elements are already implemented and some others will be added in the Rust version. The others are not in the spirit of the project
